app/partials/header.html

```diff
@@ -1,8 +1,22 @@
 <div class="header">
-  <ul class="nav nav-pills pull-right">
-    <li class="active"><a href="#">Home</a></li>
-    <li><a href="#">About</a></li>
-    <li><a href="#">Contact</a></li>
-  </ul>
-  <h3 class="text-muted">PageTraining</h3>
+  <div class="row head">
+    <div class="col-md-1 logo"></div>
+    <div class="col-md-4 col-md-offset-3"><input type="text" class="search"></div>
+    <div class="col-md-1 col-md-offset-3 price"></div>
+  </div>
+
+
+  <div class="row uppercase" id="menu">
+    <div class="col-md-offset-1 col-md-1"><p>Home</p></div>
+    <div class="col-md-1"><p>Sale</p></div>
+    <div class="col-md-1"><p>Handbag</p></div>
+    <div class="col-md-1"><p>Wallet</p></div>
+    <div class="col-md-1"><p>Accesories</p></div>
+    <div class="col-md-1"><p>Men Store</p></div>
+    <div class="col-md-1"><p>Shoes</p></div>
+    <div class="col-md-1"><p>Vintage</p></div>
+    <div class="col-md-1"><p>Services</p></div>
+    <div class="col-md-1"><p>Contact Us</p></div>
+  </div>
```

Comment:
All menu items should be links. It's ok to be inside a div, but instead of a `<p>` you should use an `<a>`